app-emulation/firecracker-bin/firecracker-bin-0.25.1.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit linux-info
+
+DESCRIPTION="Secure and fast microVMs for serverless computing (static build)"
+HOMEPAGE="https://firecracker-microvm.github.io https://github.com/firecracker-microvm/firecracker"
+SRC_URI="
+	amd64? (
+		https://github.com/firecracker-microvm/firecracker/releases/download/v${PV}/firecracker-v${PV}-x86_64.tgz
+	)
+	arm64? (
+		https://github.com/firecracker-microvm/firecracker/releases/download/v${PV}/firecracker-v${PV}-aarch64.tgz
+	)"
+
+LICENSE="|| ( Apache-2.0 MIT Apache-2.0-with-LLVM-exceptions ) MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64"
+
+RESTRICT="test strip"
+
+RDEPEND="!app-emulation/firecracker
+	acct-group/kvm"
+
+QA_PREBUILT="/usr/bin/firecracker
+	/usr/bin/jailer"
+
+S="${WORKDIR}"
+
+pkg_pretend() {
+	if use kernel_linux && kernel_is lt 4 14; then
+		eerror "Firecracker requires a host kernel of 4.14 or higher."
+	elif use kernel_linux; then
+		if ! linux_config_exists; then
+			eerror "Unable to check your kernel for KVM support"
+		else
+			CONFIG_CHECK="~KVM ~TUN ~BRIDGE"
+			CONFIG_CHECK="~KVM ~TUN ~BRIDGE"
+			ERROR_KVM_AMD="If you have an AMD CPU, you must enable KVM_AMD in"
```

Comment:
Believe accidentally deleted `ERROR_KVM=` instead of the duplicate here, I'll restore it to match other ebuilds.

Reply:
Thank you for fixing this.